recipes/pulpy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pulpy" %}
+{% set version = "1.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3036611410c6a1213d20ae7512cc00444d402d761f31f593e506144b1e0a3a74
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
```

Comment:
Under `host`, typically we just need `pip`, `python` (with the version constraint) and `setuptools`. I don't think the rest are required to build the package. 

Additionally in the `pyproject.toml` on pypi, it looks like this supports `python >=3.8`

Reply:
This has been resolved! Requirements everywhere are now python >=3.9